lib/logflare_web/controllers/plugs/set_verify_source.ex

```diff
@@ -14,76 +16,22 @@ defmodule LogflareWeb.Plugs.SetVerifySource do
 
   def call(%{assigns: %{user: user}, params: params} = conn, _opts) do
     id = params["source_id"] || params["id"]
-    token = params["source"] || params["source_id"]
-    name = params["source_name"]
-
-    is_api_path = conn.request_path =~ "/logs" or conn.request_path =~ "/api"
-    is_browser_path = not is_api_path
-
-    token =
-      if is_browser_path || Sources.valid_source_token_param?(token) do
-        token
-      else
-        nil
-      end
-
-    source =
-      cond do
-        token && is_api_path ->
-          Sources.Cache.get_by_and_preload_rules(token: token)
-          |> Sources.refresh_source_metrics_for_ingest()
-
-        name && is_api_path ->
-          Sources.Cache.get_by_and_preload_rules(name: name, user_id: user.id)
-          |> Sources.refresh_source_metrics_for_ingest()
-
-        id && is_browser_path ->
-          Sources.get_by_and_preload(id: id)
-
-        name && is_browser_path ->
-          Sources.get_by_and_preload(name: name)
-
-        true ->
-          nil
-      end
-
+    source = Sources.get_by_and_preload(id: id)
     user_authorized? = &(&1.user_id === user.id || user.admin)
```

Comment:
Why have this as lambda instead of function or just plain variable? I see it is used only once.

Reply:
will remove.